test/dlc_tests/sanity/test_security_check.py

```diff
@@ -25,7 +26,7 @@
 )
 from src.config import is_ecr_scan_allowlist_feature_enabled
 
-LOWER_THRESHOLD_IMAGES = {"mxnet": "1.8.0"}
+LOWER_THRESHOLD_IMAGES = {"mxnet":SpecifierSet(">=1.8.0,<1.9.0")}
```

Comment:
is this global used anywhere outside this file?

Reply:
No, it is not used anywhere outside the file.

Reply:
nit: this is now lower and upper threshold. do you want to change the name?

Reply:
style nit: can you add space after ":"